ios-quickeditor/ios-quickeditor/QEFilesListViewController.m

```diff
@@ -41,16 +36,14 @@ - (void)awakeFromNib
 
 - (void)viewDidLoad
 {
-    [super viewDidLoad];
-    
-    // Check for authorization.
-    GTMOAuth2Authentication *auth =
-    [GTMOAuth2ViewControllerTouch authForGoogleFromKeychainForName:kKeychainItemName
-                                                          clientID:kClientId
-                                                      clientSecret:kClientSecret];
-    if ([auth canAuthorize]) {
-        [self isAuthorizedWithAuthentication:auth];
-    }
+    // Configure Google Sign-in.
+    GIDSignIn* signIn = [GIDSignIn sharedInstance];
+    signIn.delegate = self;
```

Comment:
The Google Sign-in quickstart has the AppDelegate as the signIn.delegate, and this may be a best practice. I didn't do this for the iOS quickstarts in an attempt to simplify things, but it may be worth doing it here.